src/site/content/en/blog/coop-coep/index.md

```diff
@@ -234,28 +253,36 @@ The parameters object has three properties:
 
 The `group` property names your various reporting endpoints. Use these names to
 direct a subset of your reports. For instance, in the
-`Cross-Origin-Embedder-Policy` directive you can specify the relevant endpoint
-by providing the group name to `report-to=`. For example:
+`Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy` directives you
+can specify the relevant endpoint by providing the group name to `report-to=`.
+For example:
 
 ```http
-Cross-Origin-Embedder-Policy: require-corp; report-to="coep_rollout_1"
+Cross-Origin-Embedder-Policy: require-corp; report-to="coep_report"
+Cross-Origin-Opener-Policy: same-origin; report-to="coop_report"
 ```
 When the browser encounters this, it will cross reference the `report-to` value
 with the `group` property on the `Report-To` header to look up the endpoint.
-This example cross references on `coep_rollout_1` to find the endpoint
-`https://first-party-test.glitch.me/report`.
+This example cross references on both `coep_report` and `coop_report` to find
+the endpoint `https://first-party-test.glitch.me/report`.
 
-If you prefer to receive reports without blocking any embedded content, use
-`Cross-Origin-Embedder-Policy-Report-Only` instead of
-`Cross-Origin-Embedder-Policy`. For example:
+If you prefer to receive reports without blocking any embedded content or
+isolate a popup window, append `-Report-Only` to respective headers: i.e.
+`Cross-Origin-Embedder-Policy-Report-Only` and
+`Cross-Origin-Opener-Policy-Report-Only`. For example:
 
 ```http
-Cross-Origin-Embedder-Policy-Report-Only: require-corp; report-to="coep_rollout_1"
+Cross-Origin-Embedder-Policy-Report-Only: require-corp; report-to="coep_report"
+Cross-Origin-Opener-Policy-Report-Only: same-origin; report-to="coop_report"
 ```
 
 By doing this, when the browser detects cross origin resources that don't have
 CORP or CORS, it sends a report using the Reporting API without actually
-blocking those resources.
+blocking those resources because of COEP.
+
+Similarly, when the browser opens a cross-origin popup window, it sends a report
+without actually isolating the window because of COOP. When on report-only mode,
+sending and receiving a `postMessage()` also emit a report.
```

Comment:
I understand this as postMessage being the only cross window access that can trigger a report, where in fact most of them would. (Camille mentions this in her last comment)

Reply:
@ParisMeuleman Looks like you are looking at an old commit. Can you take a look at this one instead? https://github.com/GoogleChrome/web.dev/pull/3797/files#diff-319af7efaafc2eb017481440c77c25eaR339

Reply:
Thanks Eiji, LGTM